<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CerebrumExtractionDialog</class>
 <widget class="QDialog" name="CerebrumExtractionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cerebrum Extraction</string>
  </property>
  <widget class="QPushButton" name="previousStepButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runStepButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Step &gt;</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="nextTool">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Stage: Cortex Identification</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cerebrum Labeling</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="stepNameText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Initialization</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linearConvergenceEditBox">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>180</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear Convergence</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Warp Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>242</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost Function</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="warpConvergenceEditBox">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Warp Convergence</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="warpLevelSpinBox">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="costFunctionComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>239</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="verboseCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Verbose</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showAlignedAtlasAsOverlayCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Show aligned atlas as overlay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="atlasPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Atlas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="labelsPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Labels</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tempPathPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Temp</string>
   </property>
  </widget>
  <widget class="DSFileLabel" name="atlasFileTextLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="DSFileLabel" name="labelFileTextLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="DSFileLabel" name="tempPathTextLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="statusBoxTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreFromPreviousSessionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore From Previous...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="useCentroidsCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialize using centroids</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DSFileLabel</class>
   <extends>QLabel</extends>
   <header>dsfilelabel.h</header>
   <slots>
    <signal>fileDropped(QString)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
